code/game/gamemodes/cult/talisman.dm

```diff
@@ -1,255 +1,404 @@
+	. = ..()
+	var/mob/living/carbon/human/H = user
+	user.visible_message("<span class='warning'>Otherworldly armor suddenly appears on [user]!</span>", \
+						 "<span class='cultitalic'>You speak the words of the talisman, arming yourself!</span>")
+	H.equip_or_collect(new /obj/item/clothing/suit/hooded/cultrobes/alt(user), slot_wear_suit)
+	H.equip_or_collect(new /obj/item/weapon/storage/backpack/cultpack(user), slot_back)
+	H.drop_item()
+	H.put_in_hands(new /obj/item/weapon/melee/cultblade(user))
+	H.put_in_hands(new /obj/item/weapon/restraints/legcuffs/bola/cult(user))
+
+/obj/item/weapon/paper/talisman/armor/attack(mob/living/target, mob/living/user)
+	if(iscultist(user) && iscultist(target))
+		user.drop_item()
+		invoke(target)
+		qdel(src)
+		return
+	..()
+
+
+//Talisman of Horrors: Breaks the mind of the victim with nightmarish hallucinations
+/obj/item/weapon/paper/talisman/horror
+	cultist_name = "Talisman of Horrors"
+	cultist_desc = "A talisman that will break the mind of the victim with nightmarish hallucinations."
+	invocation = "Lo'Nab Na'Dm!"
+
+/obj/item/weapon/paper/talisman/horror/attack(mob/living/target, mob/living/user)
+	if(iscultist(user))
+		to_chat(user, "<span class='cultitalic'>You disturb [target] with visons of the end!</span>")
+		if(iscarbon(target))
+			var/mob/living/carbon/H = target
+			H.AdjustHallucinate(30)
+		qdel(src)
+
+
+//Talisman of Fabrication: Creates a construct shell out of 25 metal sheets.
+/obj/item/weapon/paper/talisman/construction
+	cultist_name = "Talisman of Construction"
+	cultist_desc = "Use this talisman on at least twenty-five metal sheets to create an empty construct shell or on plasteel to make runed metal"
+	invocation = "Ethra p'ni dedol!"
+	uses = 25
+
+/obj/item/weapon/paper/talisman/construction/attack_self(mob/living/user)
+	if(iscultist(user))
+		to_chat(user, "<span class='warning'>To use this talisman, place it upon a stack of metal sheets or plasteel sheets!.</span>")
+	else
+		to_chat(user, "<span class='warning'>You see strange symbols on the paper. Are they supposed to mean something?</span>")
+
+
+/obj/item/weapon/paper/talisman/construction/attack(obj/M,mob/living/user)
+	if(iscultist(user))
+		to_chat(user, "<span class='cultitalic'>This talisman will only work on a stack of metal sheets or plasteel sheets!!</span>")
+		log_game("Construct talisman failed - not a valid target")
+
+/obj/item/weapon/paper/talisman/construction/afterattack(obj/item/stack/sheet/target, mob/user, proximity_flag, click_parameters)
+	..()
+	if(proximity_flag && iscultist(user))
+		if(istype(target, /obj/item/stack/sheet/metal))
+			var/turf/T = get_turf(target)
+			if(target.use(25))
+				new /obj/structure/constructshell(T)
+				to_chat(user, "<span class='warning'>The talisman clings to the metal and twists it into a construct shell!</span>")
+				user << sound('sound/magic/Staff_Chaos.ogg',0,1,25)
+				qdel(src)

```

Comment:
This needs to `return` - otherwise, you'll still get the "metal or plasteel" message